### DISCUSSION TITLE
Column-level access control

### DISCUSSION THREAD

**Discussion by phuoccss** [supabase/supabase#1180] [Questions] [answered]
Hi, I want to limit update to certain columns in a table. How may I achieve this?

I've tried
```sql
revoke update on projects from authenticated;
```

and also 
```sql
grant
  update (column)
on table to authenticated;
```

It doesn't seem to work.

**Comment by steve-chavez** [comment, marked as answer]
> It doesn't seem to work.

What's the error message? Can you share a reproducible test case?

**Reply by phuoccss** [reply]
What happened was I could still update the non-writable columns. Anw, I dropped the table and started from scratch and it's working fine now. My bad, first time learning Postgres 😀
